internal/conduit/pipeline/eth_client.go

```diff
@@ -0,0 +1,38 @@
+package pipeline
+
```

Comment:
Let's create a low priority issue for a RetryableEthClient

Reply:
Also introduce idiomatic constructor patterns as well

Reply:
Created #20